internal/storage/image_test.go

```diff
@@ -604,7 +604,7 @@ var _ = t.Describe("Image", func() {
 			Expect(err).To(BeNil())
 
 			// When
-			res, err := sut.PullImage(imageRef, &storage.ImageCopyOptions{
+			res, err := sut.PullImage(context.Background(), imageRef, &storage.ImageCopyOptions{
```

Comment:
should there be a test here to prove the pull can cancel on context?

Reply:
Sure, added a test with a cancelled context.